src/applications/gi/containers/GiBillApp.jsx

```diff
@@ -105,10 +103,7 @@ export class GiBillApp extends React.Component {
               facilityCode={facilityCode}
               location={this.props.location}
             />
-            <DowntimeNotification
-              appTitle={'GI Bill Comparison Tool'}
-              dependencies={[externalServices.global]}
-            >
+            <DowntimeNotification appTitle={'GI Bill Comparison Tool'}>
```

Comment:
I'm guessing we want to account for global downtime with GIBCT, which is why I left the DTN here without any dependencies.

But why isn't there a dependency for it as an individual app? Is that dependency GIDS, and is there a PagerDuty service that corresponds to it?

Reply:
There is a "Component: GI Bill Comparison Tool (GIBCT|GIDS)" service in PagerDuty that doesn't get propagated down from the API as one of the services that have statuses.

Reply:
i think we just document how to set up DTN and let the education team know about it. AFAIK we wan't tracking this yet.